components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -641,7 +641,10 @@ export default class RhsRootPost extends React.PureComponent {
                             {fileAttachment}
                             <div className='post__body-reactions-acks'>
                                 {this.props.isPostAcknowledgementsEnabled && post.metadata?.priority?.requested_ack && (
-                                    <PostAcknowledgements postId={post.id}/>
+                                    <PostAcknowledgements
```

Comment:
0/5 - I'm seeing this pattern a few times of checking if post acknowledgement is enabled, if the post requires acknowledgement, and then checking if the post is deleted in the other cases. It might be nice if the `PostAcknowledgements` component could figure that out itself so we just always render it and let the component decide whether or not to render as null

Reply:
@hmhealey the only issue I have with going that way is that, hooks, and `mapStateToProps`, have to run before any conditional return, so we'll be executing code for each post that we won't need to otherwise, since posts that have ACKs requested are going to be a small subset of rendered posts, if any.
I'm not sure if that would have any noticeable effect on performance though, I just wanted to be on the "safe" side.

Reply:
I'm gonna move the "is deleted" clause into the component as a `isDeleted` prop, just to force users to not forget that one.